src/types/player.ts

```diff
@@ -0,0 +1,13 @@
+import { PluginType } from './plugin';
+
+export interface Player {
```

Comment:
This should extend the `Plugin` type. :slightly_smiling_face:

Reply:
Fixed